docs-src/traffic-shifting.md

```diff
@@ -0,0 +1,97 @@
+# HTTP traffic shifting
+
+The [HTTPRoute resource](httproute.md) allows you to specify weights to shift
+traffic between different backends. This is useful for shifting traffic during
+rollouts, canarying changes, or for emergencies. The HTTPRoute
+`spec.rules.forwardTo` accepts a list of backends that a route rule will send
+traffic to. The relative weights of these backends define the split of traffic
+between them. The following YAML snippet shows how two Services are listed as
+backends for a single route rule. This route rule will split traffic 90% to
+`foo-svc-1` and 10% to `foo-svc-2`.
+
+![Traffic shifting](./images/traffic-shifting-2.png)
+
+```
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: traffic-split
+spec:
+  rules:
+  - forwardTo:
+    - serviceName: foo-svc-1
+      port: 8080
+      weight: 90
+    - serviceName: foo-svc-2
+      port: 8080
+      weight: 10
+...
```

Comment:
What do the dots here represent? I'm probably forgetting something obvious, but would this be a valid route as is?

Reply:
I think it is best to `include` a valid HTTPRoute resource here. It is verbose but allows users to copy-paste and hit the ground running.

Reply:
okay I added in the full resource with an include